awx_collection/plugins/doc_fragments/auth_plugin.py

```diff
@@ -17,45 +17,51 @@ class ModuleDocFragment(object):
     description: The network address of your Automation Platform Controller host.
     env:
     - name: CONTROLLER_HOST
-    - name: TOWER_HOST
       deprecated:
         collection_name: 'awx.awx'
         version: '4.0.0'
         why: Collection name change
-        alternatives: 'CONTROLLER_HOST'
+        alternatives: 'TOWER_HOST, AAP_HOSTNAME'
   username:
     description: The user that you plan to use to access inventories on the controller.
     env:
     - name: CONTROLLER_USERNAME
-    - name: TOWER_USERNAME
       deprecated:
         collection_name: 'awx.awx'
         version: '4.0.0'
         why: Collection name change
-        alternatives: 'CONTROLLER_USERNAME'
+        alternatives: 'TOWER_USERNAME, AAP_USERNAME'
   password:
     description: The password for your controller user.
     env:
     - name: CONTROLLER_PASSWORD
-    - name: TOWER_PASSWORD
       deprecated:
         collection_name: 'awx.awx'
         version: '4.0.0'
         why: Collection name change
-        alternatives: 'CONTROLLER_PASSWORD'
+        alternatives: 'TOWER_PASSWORD, AAP_PASSWORD'
+  oauth_token:
+    description:
+    - The OAuth token to use.
+    env:
+    - name: CONTROLLER_OAUTH_TOKEN
+      deprecated:
+        collection_name: 'awx.awx'
+        version: '4.0.0'
+        why: Collection name change
+        alternatives: 'TOWER_OAUTH_TOKEN, AAP_TOKEN'
```

Comment:
Same with this. Not sure how to handle this so it is clear that OAUTH is coming from Gateway but is not available in Controller